CODEOWNERS

```diff
@@ -4,24 +4,71 @@
 
 # Each line is a file pattern followed by one or more owners.
 # Please feel free to add yourself to a module or create a new mapping.
+# Ideally each module should have two or more owners.
 
-# These owners will be the default owners for everything in the repo.
-# Later match takes precedence
+# Code owners are automatically requested for review
+# when someone opens a pull request that modifies code that they own.
+# Later matches take precedence.
 
-std/bitmanip.d @WalterBright
-std/compiler.d @WalterBright
+etc/c/* @CyberShadow
+std/algorithm/* @andralex @wilzbach
+#std/array.d
+#std/ascii.d
+#std/base64.d
+#std/bigint.d
+#std/bitmanip.d
+std/c/windows/* @CyberShadow
+#std/compiler.d
+#std/complex.d
 std/concurrency.d @MartinNowak
+#std/container/
+std/conv.d @JackStouffer
+#std/csv.d
 std/datetime/* @jmdavis
+#std/demangle.d
 std/digest/* @jpf191
+#std/encoding.d
+#std/exception.d
 std/experimental/allocator/* @andralex
 std/experimental/checkedint/* @andralex
 std/experimental/logger/* @burner
+#std/experimental/typecons.d
+#std/file.d
+#std/format.d
+#std/functional.d
+#std/getopt.d
+#std/internal/
+std/json.d @CyberShadow
 std/math* @don-clugston-sociomantic @WalterBright
+#std/math.d
+#std/mathspecial.d
 std/meta.d @klickverbot @ZombineDev
+#std/mmfile.d
+std/net/curl.d @CyberShadow
+#std/net/isemail.d
+#std/numeric.d
+#std/outbuffer.d
+#std/parallelism.d
+std/path.d @CyberShadow
 std/process.d @kyllingstad @schveiguy @CyberShadow
 std/random.d @WebDrake @wilzbach
+std/range/* @andralex @jmdavis @wilzbach
```

Comment:
jmdavis has remarked on how busy he is lately. I suggest you take him off because I don't he has much time to even respond to datetime changes.

You can put me here instead